website/content/docs/templates/hcl_templates/locals.mdx

```diff
@@ -1,35 +1,34 @@
 ---
-page_title: Local Values - HCL Configuration Language
+page_title: Local Variables - HCL Configuration Language
 description: >-
-  Local values assign a name to an expression that can then be used multiple
+  Local variables assign a name to an expression that can then be used multiple
```

Comment:
I'm not sure if descriptions should be active or passive voice so feel free to disregard if this is wrong:
```suggestion
  Local variables assign a name to an expression that you can use multiple
```